### DISCUSSION TITLE
Is there a way to indicate a specific text for a cross reference?

### DISCUSSION THREAD

**Discussion by Pierre9344** [quarto-dev/quarto-cli#10057] [Q&A] [answered]
### Description

Hello,
Is there a way to specify a specific text for a cross reference? I have a file that correspond to a figures with 3 sub figures.
In my text, I want to make a reference to a specific panel of the figure. I currently do it as bellow:
```
::: {#fig-brain-anat}

![](./images/CNS.jpeg)

Anatomie du système nerveux central. Localisation du SNC (encéphale et moelle épinière) dans le corps humain (A), représentation graphique de l'encéphale (B), et représentation graphique de la barrière hémato-encéphalique (C).
:::

Please refer to @fig-brain-anat C
```

The problem is that in the rendered document, I got something like "Please refer to Figure 1 C" and only the "Figure 1" part have an hyperlink to the figure. What I would like is to force the text to: "Please refer to Figure 1C" with the entire "Figure 1C" part being the hyperlink.

Is it currently possible?

**Comment by mcanouil** [comment, marked as answer]
No, it's not currently possible, see <https://quarto.org/docs/authoring/cross-references.html#references>.

You might want to use separate figures: https://quarto.org/docs/authoring/cross-references.html#subfigures

**Reply by Pierre9344** [reply]
Thank you for your answer.

Using multiple figures is not good in my case because because some of my subfigures are relied to each other with arrow. I think I will manually correct that in the rendered document.
